components/builder-web/app/shared/statuspage/_statuspage-indicator.component.scss

```diff
@@ -0,0 +1,25 @@
+// The CSS for the colored dot that represents your current status
+$red:#e74c3c;
+$orange:#e67e22;
+$yellow:#f1c40f;
+$green:#2ecc71;
+
+.color-dot {
+  border-radius: 99px;
+  display:inline-block;
```

Comment:
We don't have a lint rule for it yet, but there should be a space after the `:` for these.